Jenkinsfile

```diff
@@ -63,7 +63,7 @@ pipeline {
             }
           }
           stages {
-            stage('Checkout code') { steps { dir('k-exercises') { git url: 'git@github.com:kframework/k-exercises.git', credentialsId: 'rv-jenkins-access-token' } } }
+            stage('Checkout code') { steps { dir('k-exercises') { git url: 'git@github.com:kframework/k-exercises.git' } } }
```

Comment:
This is just undoing a previous thing that did _not_ correctly solve a credentials issue. @dwightguth you'll have to review this one.